_posts/2014-02-25-omero-bio-formats-5-0-0.md

```diff
@@ -64,7 +64,7 @@ For OMERO:
 - CLI plugin allowing access to tags and tag sets from the [command line](https://www.openmicroscopy.org/site/support/omero5.0/users/command-line-interface.html)
 - Modulo (FLIM/SPIM) support in OMERO.insight
 - Many bug fixes including thumbnails, plates, tables and LDAP
-- New help guide for installing the plugins needed to use [OMERO with ImageJ](https://help.openmicroscopy.org/imagej.html)
+- New help guide for installing the plugins needed to use [OMERO with ImageJ](https://omero-guides.readthedocs.io/en/latest/fiji/docs/index.html)
```

Comment:
Minor comment: i am not sure we want to edit historical timestamped announcement posts esp. when we control these URLs and plan to remap them.

Reply:
we will redirecting. I updated them since there were only a few to adjust

Reply:
removed commit from the git history